lib/matplotlib/tests/test_backend_pgf.py

```diff
@@ -276,3 +277,14 @@ def test_pdf_pages_lualatex():
         pdf.savefig(fig)
 
         assert pdf.get_pagecount() == 2
+
+
+def test_tex_restart_after_error():
```

Comment:
Probably need to add
```python
@needs_xelatex
@pytest.mark.backend('pgf')
```
?

Reply:
I think only the first one (the backend will be autoswitched anyways), thanks.